### PR TITLE
Issue 1.12 clickable cohort list

### PR DESCRIPTION
**Pull request by martapa**
Clickable cohorts list.
 
I added two very basic tests. 

image

Also, I think prettier doesn't work properly for me. We expect single quotes?

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,12 +3,16 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "fsevents": "^2.1.2",
```
Comment (IrinaGM) [inline package.json]: this dependancy was causing us issues with netlify. can you please remove it from package.json?

File: src/App.js [hunk 2]
```diff
@@ -1,32 +1,31 @@
 import React, { useEffect } from "react";
-import logo from "./logo.svg";
-import "./App.css";
+import "./App.scss";
```
Comment (IrinaGM) [inline src/App.js]: I think for this one you would probably need to pull my branch again to see what are the final changes.

File: src/App.scss [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+.App {
+  font: 100% $font-stack;
+  color: $primary-color;
+}
+
+.header {
+  width: 100%;
+  height: 8vh;
+}
```
Comment (IrinaGM) [inline src/App.scss]: Same here, as I've removed this file, probably best to pull the changes from my branch and see if there are nay conflicts.

**Comment by vindhya** [comment]
> Clickable cohorts list.
> 
> I added two very basic tests.
> 
> image
> 
> Also, I think prettier doesn't work properly for me. We expect single quotes?

Looks good! Thanks Marta!

In terms of prettier, yes, we're expecting single quotes in js files. Do you have the prettier extension installed in VS Code?

File: src/components/CohortList/CohortListStyle.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import styled from "styled-components";
+
+export const List = styled.ul`
```
Comment (IrinaGM) [inline src/components/CohortList/CohortListStyle.js]: General question: do we want to call the styled components that we are exporting with "Style" at the end or not? @vindhya @martapa @yuliwna1

Comment (vindhya) [reply src/components/CohortList/CohortListStyle.js]: I agree that the naming should be consistent if it's a component being used purely for style. I'm good with the "Style" suffix at the end.

After doing a little googling, I've seen people using the prefix "Styled" for the styled-components, like "StyledList". Also seen people using "Wrapper" as a suffix like "ListWrapper".

**Pull request merged**